### PR TITLE
simpler installation process

### PR DESCRIPTION
**Pull request by m-beau**
Phy and phycontrib themselves are being installed through the environment file.

In the end, the use simply has to do: conda create -f path/to/environment.yml, and he is setup!

WARNING the @dev branches are used here, this will have to be changed.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -16,3 +16,5 @@ dependencies:
     - tqdm
     - joblib
     - click
+    - git+https://github.com/cortex-lab/phy.git@dev
+    - git+https://github.com/cortex-lab/phy-contrib.git@dev
```
Comment (rossant) [inline environment.yml]: to replace with `git+https://github.com/cortex-lab/phylib.git`
phy-contrib is deprecated in phy2

Comment (m-beau) [reply environment.yml]: Done

Comment (rossant) [reply environment.yml]: thanks but the first line (phy) was good!

**Comment by rossant** [comment]
looks good!
the `.travis.yml` file should also be updated for continuous integration

- line 21: `- source activate phy2`
- remove line 26

I think that's all

**Comment by rossant** [comment]
thanks! integrated in dev

**Pull request closed without merging**